src/components/inputs/rich-text-input/rich-text-input.js

```diff
@@ -8,13 +9,30 @@ import plugins from '../../internals/rich-text-plugins';
 import html from '../../internals/rich-text-utils/html';
 import isEmpty from '../../internals/rich-text-utils/is-empty';
 
-class RichTextInput extends React.PureComponent {
+const removeProps = ['value'];
+
+class RichTextInput extends React.Component {
   state = {
     // we keep track of the serialized (HTML) value
     serializedValue: this.props.value || '',
     value: html.deserialize(this.props.value || ''),
   };
 
+  shouldComponentUpdate(nextProps, nextState) {
+    // ignore updates for changes to `props.value`
+    const props = omit(this.props, removeProps);
+
+    const havePropsChanged = Object.entries(props).some(
+      ([key, val]) => nextProps[key] !== val
+    );
+
+    if (havePropsChanged) return true;
+
+    if (this.state.value !== nextState.value) return true;
```

Comment:
I am confused but I presume it's just me. We omit `value` and if anything else changes want to render. THen we check if value changed and if so render. Isn't that the same as just not omitting `value` in the first place?

Reply:
the `prop` value is actually a serializedValue. We just call it `value` so we don't expose to the consumer that under the hood we don't deal with HTML values.

If update if the state `value` changes, which is a `slate` value 😄

Reply:
It would probably be clearer if we called `value` something else, but then it makes it weird for the consumer who doesn't really care about slate vs draftjs vs whatever...

Reply:
It's not about the value. It's about the code no matter what value is so me it reads as

1. Exclude `value`: if anything else changed then render
2. If `value` changed then render

Which is the same as not excluding it in the first plae.

Reply:
1. Exclude `props.value`. 
2. If `state.value` changes then render. Not `props.value`

Ignore props.value. Clear?